.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": false
+  },
+}
```

Comment:
was this done, for coding purposes?

Reply:
Yes, kinda. My eslint vscode extension fixes all errors on save, but when I want to produce an error in a file, to test it, ESLint autofixed it. So, basically I wasn't able to produce errors I wanted to test. 😸

Reply:
Just delete the file and the PR is approved!